test/core/models/StanzaHelper.scala

```diff
@@ -47,18 +47,29 @@ trait StanzaHelper extends TestConstants {
     Phrase(Vector("Oranges", "Welsh: Oranges"))
   )
 
-  val multipleExclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
+  val exclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
     Phrase(Vector("What kind of fruit do you like?", "Welsh: What kind of fruit do you like?")),
     Phrase(Vector(
-      "Other [exclusive][hint:Selecting this checkbox will deselect the other checkboxes]",
-      "Welsh: Other [exclusive][hint:Welsh: Selecting this checkbox will deselect the other checkboxes]")),
+      "Other [hint:Selecting this checkbox will deselect the other checkboxes]",
+      "Welsh: Other [hint:Welsh: Selecting this checkbox will deselect the other checkboxes]"
+    )),
     Phrase(Vector("Apples", "Welsh: Apples")),
-    Phrase(Vector(
-      "Bananas [exclusive][hint:Selecting this checkbox will deselect the other checkboxes]",
-      "Welsh: Bananas [exclusive][hint:Welsh: Selecting this checkbox will deselect the other checkboxes]")),
+    Phrase(Vector("Bananas", "Welsh: Bananas")),
     Phrase(Vector("Oranges", "Welsh: Oranges"))
   )
 
+  // val multipleExclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
```

Comment:
Take it the above is a refactored version of these commented out lines ? If so do we need to retain  these?

Reply:
yes, will remove

Reply:
Verified as removed